packages/graph-explorer/src/modules/KeywordSearch/KeywordSearch.tsx

```diff
@@ -413,29 +414,24 @@ const KeywordSearch = ({
             <Checkbox
               isSelected={neighborsLimit}
               onChange={onNeighborsLimitChange}
+              className={pfx("refuse-shrink")}
             >
-              <div className={pfx("neighbors-limit-checkbox")}>
-                Limit Neighbors?
-              </div>
+              Limit Neighbors?
             </Checkbox>
-            <div>
-              <IconButton
-                className={pfx("actions-button")}
-                icon={<RemoveIcon />}
-                onPress={() => selection.clear()}
-              >
-                <div className={pfx("icon-button-name")}>Clear Selection</div>
-              </IconButton>
-              <IconButton
-                className={pfx("actions-button")}
-                icon={<AddCircleIcon />}
-                onPress={handleAddEntities}
-              >
-                <div className={pfx("icon-button-name")}>
-                  {addSelectedNodesMessage()}
-                </div>
-              </IconButton>
-            </div>
+            <Button
```

Comment:
This might just be a personal preference, but I think having "clear selected" with a different look (either leave as a different type of button or give it a different color) would make it more distinguishable from the "add selection" visually.

Reply:
In general, I agree. My goal here was to fix the bugs and keep the visual changes very minor.

I'd like to save this type of change for a larger UI refresh of the search UI when I better understand how the UI is used by users.

For example, a few options for what you are suggesting could be:

- Show the "clear selection" button as secondary variant, "add selected" button as primary variant
- Show the "clear selection" button as dangerous variant (red), "add selected" button as primary variant
- Show the "clear selection" button as tertiary variant, "add selected" button as secondary variant

I'm not even sure if these variants exist in the project. So I still need to do a full audit of the themed component before I could make a determination of what option is best for this UI.